### PR TITLE
Fix can-reach format.

### LINKED ISSUE #502: IP Autodetection methods section formatting weird
http://docs.projectcalico.org/master/reference/node/configuration#ip-autodetection-methods

This section is not formatting properly.

### PR DESCRIPTION
**Pull request by djosborne**
Fixes #502

### CODE REVIEW

File: master/reference/node/configuration.md [hunk 1]
```diff
@@ -49,17 +49,20 @@ very simplified guess, it is recommended to either configure the node with a
 specific IP address, or to use one of the other detection methods.
 
 e.g.
+
```
Comment (robbrockbank) [inline master/reference/node/configuration.md]: Should probably add a blank line below the first-found heading too then for completeness no?

**Comment by robbrockbank** [comment]
Looks like the same changes are required for the `calicoctl node run` command.

**Comment by robbrockbank** [comment]
oops closed accidentally... reopened.

**Comment by djosborne** [comment]
Fixed the `node run` ones too.

**Comment by robbrockbank** [comment]
lgtm

**Pull request merged**